packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -493,14 +504,15 @@ export interface ColumnProps extends RACColumnProps {
 /**
  * A column within a `<Table>`.
  */
-export function Column(props: ColumnProps) {
+export const Column = forwardRef(function Column(props: ColumnProps, ref: DOMRef<HTMLDivElement>) {
```

Comment:
these exports don't have the same `/*#__PURE__*/` export since they don't have generics in their type. The ones that do need `/*#__PURE__*/` for treeshaking